danger-sample/danger-sample/ViewController.swift

```diff
@@ -20,6 +20,5 @@ class ViewController: UIViewController {
         // Dispose of any resources that can be recreated.
     }
 
-
```

Comment:
Did you mean to leave this deletion in? Running `swiftlint` manually doesn't produce any warnings for this file without it.

Reply:
Are you running on xcode 9? There are at least 2 more warnings in the project:
![appdelegate swift 2017-10-07 23-30-15](https://user-images.githubusercontent.com/2203199/31311554-88849840-abb7-11e7-8c74-588e10d0f6a6.jpg)

Reply:
Ah, `danger-swiftlint` only lints the files that changed in the pull request, so it wouldn't be listing this file.

Reply:
Is there a way to override that? Every swiftlint release comes with some new rules, so older files might be affected too

Reply:
Not really, `danger-swiftlint` is specifically designed to check only the added/modified files.

Reply:
Not sure if this is a desired behaviour. Swiftlint might be upgraded in CI independently. A new warnings may appear. 
If `danger-swiftlint` will remain working in this mode, then may we have a separate filelist option?

Reply:
It's possible to pin the version of swiftlint using the `SWIFTLINT_VERSION` and running the `rake install` task on CI. I'm afraid I don't think that providing a file list fits with danger-swiftlint. However, [all the code you'd need](https://github.com/ashfurrow/danger-swiftlint/blob/master/lib/danger_plugin.rb) to build this is open source, so you could add this to your Dangerfile or build a new plugin. If you have any questions, please let me know 👍